### PR TITLE
font-ubuntu-sans-ttf: Add at 1.006

### PR DESCRIPTION
**Pull request by uni-dos**
**Summary**
- Add ubuntu-sans font
- resolves https://github.com/getsolus/packages/issues/4360

**Test Plan**

Install font, and view in kde font manager.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by uni-dos** [comment]
The font needs to be added to[Anitya

**Comment by malfisya** [comment]
Added to anitya [here](https://release-monitoring.org/project/375586/). :+1:

File: packages/f/font-ubuntu-sans-ttf/package.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name       : font-ubuntu-sans-ttf
+version    : 1.006
+release    : 1
+source     :
+    - https://github.com/canonical/Ubuntu-Sans-fonts/archive/refs/tags/v1.006.tar.gz : ddd9e2b74767cf2b5b33d933ff8d579e806b0b26e9b7abf96ebfe9b3f6895a65
+homepage   : https://github.com/canonical/Ubuntu-Sans-fonts
+license    : Ubuntu Font License-1.0
```
Comment (malfisya) [inline packages/f/font-ubuntu-sans-ttf/package.yml:7]: The correct identifier is `Ubuntu-font-1.0` according to [spdx](https://spdx.org/licenses/)

**Review by malfisya — CHANGES_REQUESTED** [review]
One small nit

**Review by malfisya — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**